fmriprep/workflows/bold/resampling.py

```diff
@@ -662,7 +667,10 @@ def init_bold_fsLR_resampling_wf(
     # RibbonVolumeToSurfaceMapping.sh
     # Line 85 thru ...
     volume_to_surface = pe.Node(
-        VolumeToSurfaceMapping(method="ribbon-constrained"),
+        VolumeToSurfaceMapping(
+            method="ribbon-constrained",
+            output_weights_text="output_weights.txt",
```

Comment:
@feilong says this isn't very helpful after all. Let's remove it before release.